### PR TITLE
doc: update macos install doc

### PR DESCRIPTION
**Pull request by gnattu**
The original doc is a bit outdated. Updated to match the 10.9 instructions.

### CODE REVIEW

File: docs/general/installation/macos.md [hunk 1]
```diff
@@ -1,82 +1,101 @@
...
-# Install on MacOS
+# Install on macOS
 
 macOS Application packages and builds in TAR archive format are available [here](/downloads/macos).
 
 :::note
-Jellyfin requires macOS 10.15 or newer to run.
+Jellyfin requires macOS 12 or newer to run.
 
 :::
```
Comment (felix920506) [inline docs/general/installation/macos.md:17]: This doesn't need to be an admonition

File: docs/general/installation/macos.md [hunk 2]
```diff
@@ -1,82 +1,101 @@
...
-1. Move `/Application/jellyfin-version` folder to the Trash. Replace version with the actual version number you are trying to delete.
+2. Move `/Application/jellyfin-version` folder to the Trash. Replace version with the actual version number you are trying to delete.
+3. Delete the folder `~/.config/jellyfin/`
+4. Delete the folder `~/.local/share/jellyfin/`
 
-**Using FFmpeg with the Portable Version**
+### Using FFmpeg with the Portable Version
 
-The portable version doesn't come with FFmpeg by default, so to install FFmpeg you have three options.
+The portable version doesn't come with FFmpeg by default, so to install FFmpeg you have following options:
```
Comment (felix920506) [inline docs/general/installation/macos.md:87]:
```suggestion
The portable version doesn't come with FFmpeg by default. There are a few options for installing FFmpeg:
```

File: docs/general/installation/macos.md [hunk 3]
```diff
@@ -1,82 +1,101 @@
...
 
+- download the jellyfin-ffmpeg from [jellyfin-repo](https://repo.jellyfin.org/?path=/ffmpeg/macos) (recommended)
```
Comment (felix920506) [inline docs/general/installation/macos.md:89]:
```suggestion
- download jellyfin-ffmpeg from the [Jellyfin repo](https://repo.jellyfin.org/?path=/ffmpeg/macos) (recommended)
```

File: docs/general/installation/macos.md [hunk 4]
```diff
@@ -1,82 +1,101 @@
...
 - use the package manager homebrew by typing `brew install ffmpeg` into your Terminal ([here's how to install homebrew if you don't have it already](https://treehouse.github.io/installation-guides/mac/homebrew)
-- download the most recent static build from [this link](https://evermeet.cx/ffmpeg/get/zip) (compiled by a third party see [this page](https://evermeet.cx/ffmpeg/) for options and information), or
+- download the most recent static build from [this link](https://evermeet.cx/ffmpeg/get/zip) (compiled by a third party see [this page](https://evermeet.cx/ffmpeg/) for options and information) (Apple Silicon builds are not available from this source)
 - compile from source available from the official [website](https://ffmpeg.org/download.html)
 
-More detailed download options, documentation, and signatures can be found.
+Once downloaded, you need to do remove the quarantine flag for the `ffmpeg` and `ffprobe`.
```
Comment (felix920506) [inline docs/general/installation/macos.md:94]:
```suggestion
Once downloaded, remove the quarantine flag for the `ffmpeg` and `ffprobe`.
```

File: docs/general/installation/macos.md [hunk 5]
```diff
@@ -1,82 +1,101 @@
...
 
-If using static build, extract it to the `/Applications/` folder.
+Ensure that both `ffmpeg` and `ffprobe` are located at the same path, then execute the following command:
 
-Navigate to the Playback tab in the Dashboard and set the path to FFmpeg under FFmpeg Path.
+```shell
+cd /path/to/ffmpeg/folder
```
Comment (felix920506) [inline docs/general/installation/macos.md:99]: Tell the user to open a terminal in the path by right clicking and choose `open folder in terminal`?

Comment (felix920506) [reply docs/general/installation/macos.md:99]: Nvm that is from automator script I forgot about that

**Review by Shadowghost — APPROVED** [review]
Maybe a note on why we recommend using jellyfin-ffmepg would be useful

**Review by felix920506 — APPROVED** [review]
Some changes look like they are updated for 10.9 so I'm putting on the future release tag. LGTM otherwise.

**Pull request merged**